recipe/build.sh

```diff
@@ -64,13 +64,29 @@ else
   MPI=ON
 fi
 
+if [[ "$(uname)" == "Linux" ]]; then
+  # Flook compilation on linux needs some hacks
```

Comment:
I don't like this. Very non-standard.

I would suggest you fix the `setup.make` input file instead, via a patch.

This should also ensure that readline is captured correctly.